src/code/audio/audio_effects.c

```diff
@@ -153,17 +153,17 @@ f32 AudioEffects_GetVibratoFreqScale(VibratoState* vib) {
         }
     }
 
-    if (vib->extent == 0.0f) {
+    if (vib->depth == 0.0f) {
         return 1.0f;
     }
 
     pitchChange = AudioEffects_GetVibratoPitchChange(vib) + 32768.0f;
-    scaledExtent = vib->extent / 4096.0f;
-    extent = scaledExtent + 1.0f;
-    invExtent = 1.0f / extent;
+    scaledExtent = vib->depth / 4096.0f;
+    depth = scaledExtent + 1.0f;
+    invExtent = 1.0f / depth;
 
     // inverse linear interpolation
-    result = 1.0f / ((extent - invExtent) * pitchChange / 65536.0f + invExtent);
+    result = 1.0f / ((depth - invExtent) * pitchChange / 65536.0f + invExtent);
```

Comment:
I have a good number of these cleaned up on my total audio branch, including these ones. I'll probably do a separate cleanup sweep to avoid scope creep